M2/Macaulay2/d/actors2.dd

```diff
@@ -160,13 +160,13 @@ newClassParent(e:Expr,classs:HashTable,parent:HashTable,returned:bool):Expr := (
      when e
      is Error do e
      is o:HashTable do (
-	  if basicType == hashTableClass then (
+	  if basicType == hashTableClass || basicType == numberClass then (
```

Comment:
I'm not sure if I fully understand this change. When is `newClassParent` called?

Reply:
It's called by `new ... of ...` and `new ... of ... from ...`, and also `newClass` with 3 arguments.  So for example:

### Before
```m2
i1 : newClass(Number, Nothing, {})
stdio:1:1:(3): error: 'new' expected a type of list or hash table
```

### After
```m2
i1 : newClass(Number, Nothing, {})

o1 = <<a list>>

o1 : Number
```

Reply:
I'm a little unsure if we want to single out `numberClass` here like this. I would prefer to do this generally for any "type family", but if you prefer to do the general case in a future PR that sounds fine.

Reply:
Yeah, that definitely sounds like a good long term plan!